server/seccomp/types.go

```diff
@@ -0,0 +1,93 @@
+package seccomp
+
+// Seccomp represents the config for a seccomp profile for syscall restriction.
+type Seccomp struct {
```

Comment:
The seccomp json profile is from Docker, how would you do that? Maybe I can't see it immediately